wp/plugin/tests/unit/test_wp_unit.ml

```diff
@@ -151,6 +151,19 @@ let unit_tests = [
   "User defined sub specs comparative change 6" >: test_plugin "user_func_spec/sub_spec_6"
     sat ~script:"run_wp_6.sh";
 
+  "Ogre lifter 1" >: test_plugin "ogre_files/ogre_lifter_1"
```

Comment:
I think the path should be changed to `ogre_files` since `ogre_lifter_x` files don't exist.

Same with `run_wp_x.sh` should be changed to `run_wpx.sh`.